_includes/content/generator-create-app.html

```diff
@@ -1,4 +1,4 @@
-<div markdown="span" class="alert alert-warning" role="alert"><i class="fa fa-warning"></i>
+<div markdown="span" class="alert alert-warning " role="alert"><i class="fa fa-warning"></i>
```

Comment:
Maybe I'm missing something.. any reason why there's a trailing space for  `class="alert alert-warning "`? There are other occurrences in this PR.

Reply:
In this case, no. I had to modify a few alerts because they were giving me problems. But here there isn't one.